Patches for issue #28

yoshimoto opened the pull request:
I wrote two patches to solve issue #28. 
Could you please review/merge this?

svenstaro commented:
This seems to be quite a lot of code for basically specifying a minimum version. Now I'm not an Autotools expert but it would seem to me that perhaps we should just have the readme specify a minimum version if that means a lot less Autotools code.

Maybe you could explain the changes a bit if I misread? I can't check properly right now as I'm on mobile.

yoshimoto commented:
Sorry for my lack of explanation. 

My pull request contains the following three fixes:
1) Added version check
2) Fixed Miss-usage of AM_CFLAGS; The flags in AM_CFLAGS are not used at all.
3) Added -Werror=implicit-function-declaration

I'd like to withdraw this pull request once, and then send new pull request, which contains the above 1) only.

svenstaro commented:
Please rebase against master. I updated our rofi version to 1.5.4.
